app/e2e/tagCheckFixture.ts

```diff
@@ -0,0 +1,52 @@
+import { test as base } from "@playwright/test";
+
+export const TAG_STATE_MUTATE = "@stateMutate";
+export const TAG_DEMO_PACKETS = "@demoPackets";
+
+const isLocal = (baseURL: string) => baseURL.startsWith("http://localhost");
+const isDev = (baseURL: string) => baseURL.startsWith("https://packit-dev.dide.ic.ac.uk");
```

Comment:
I think this is broken now that the URL has changed. You would need to parse the URL, extract the domain name and check if that part ends in `packit-dev.dide.ic.ac.uk`

Reply:
Something like
```suggestion
const isDev = (baseURL: string) => new URL(baseURL).hostname.endsWith("https://packit-dev.dide.ic.ac.uk");
```